app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -440,5 +440,16 @@ data class Patient(
         AND syncStatus == 'DONE'
     """)
     abstract fun purgeDeletedPatientAfterRetentionTime(now: Instant)
+
+    @Query("""
```

Comment:
I feel like this query should be part of `BloodPressureMeasurement` DAO and not `Patient` DAO.